PrettyPrint/html_file.html

```diff
@@ -0,0 +1,66 @@
+<h2>ARM 01, 002</h2>
+<h6>metadata</h6>
+<pre><code>Primary publication: ARM 01, 002
+Author(s): Dossin, Georges
+Publication date: 1946
+Secondary publication(s): Durand, Jean-Marie, LAPO 16, 0306
+Collection: National Museum of Syria, Damascus, Syria
+Museum no.: NMSD —
+Accession no.:
+Provenience: Mari (mod. Tell Hariri)
+Excavation no.:
+Period: Old Babylonian (ca. 1900-1600 BC)
+Dates referenced:
+Object type: tablet
+Remarks:
+Material: clay
+Language: Akkadian
+Genre: Letter
+Sub-genre:
+CDLI comments:
+Catalogue source: 20050104 cdliadmin
+ATF source: cdlistaff
+Translation:
+UCLA Library ARK: 21198/zz001rsp9f
+Composite no.:
+Seal no.:
+CDLI no.: P254203
+</code></pre>
+<h6>transliteration</h6>
+<pre><code>&amp;P254203 = ARM 01, 002
+#atf: lang akk
+@tablet
+@obverse
+1. a-na ia-ah-du-[li-im]
```

Comment:
Would this make more sense as an ordered list (`<ol />`)? Or should the text be divided into `<p />` tags? I don't think <code /> is the best option here.

Reply:
See prettyprint.py comment / response for this: 

I'm using markdown.markdown; to change this, I will be making adjustments to my markdown copy unless I simply make an html print copy from scratch. What would be the right decision for this?

Reply:
I personally think `ol` is the best choice here, though `p` tags work. You're not formatting code though so the `code` tag shouldn't be used.